integration_tests/mockApis/licence.ts

```diff
@@ -1,12 +1,17 @@
 import { SuperAgentRequest } from 'superagent'
 import { stubFor } from '../wiremock'
 import LicenceStatus from '../../server/licences/licenceStatus'
-import Policy from './polices/v2-0'
+// eslint-disable-next-line camelcase
+import Policyv2_0 from './polices/v2-0'
```

Comment:
usual style guide for js is:
data/variables = lower camel case
types and classes = upper camel case

Reply:
that would remove the need for eslint wrangling

Reply:
I can fix that, but I believe it also complains about the underscore in the variable name. Not a lot I can do about that though as otherwise it would be `v20` and `v21`, which can lead to some predicable issues with future versioning!

Reply:
Unfortunately even dropping the case to make it `policyV2_x` still causes the camelcase error, so I've had to leave the eslint tomfoolery in